static/js/script.js

```diff
@@ -1,73 +0,0 @@
-function toastMessage(tag, message) {
```

Comment:
Was this entire file deleted? Do we not need this anymore?

Reply:
Just the contents. No I changed the logic of it and it is no longer needed. The problem was (at least for me) that the JS tried to prevent the default of submitting the form and then send it via AJAX, but sometimes it was not preventing it. So what I did was just change it to be posted to a view and just redirect back to the page.